project/Dependencies.scala

```diff
@@ -81,7 +81,16 @@ object Dependencies {
 
   val DiscoveryAwsApi = Seq(
     libraryDependencies ++=
-      DependencyGroups.AkkaActor ++ Seq("com.amazonaws" % "aws-java-sdk-ec2" % "1.11.257")
+      DependencyGroups.AkkaActor ++ Seq(
+        "com.amazonaws" % "aws-java-sdk-ec2" % "1.11.292",
+        "com.amazonaws" % "aws-java-sdk-ecs" % "1.11.292")
+  )
+
+  val DiscoveryAwsApiAsync = Seq(
+    libraryDependencies ++=
+      DependencyGroups.AkkaActor ++ Seq(
+        "software.amazon.awssdk" % "ecs" % "2.0.0-preview-9",
```

Comment:
Ok to use the new one but we should mention that we do so in the docs

Reply:
I included something in the main discovery docs: https://github.com/akka/akka-management/pull/140/files#diff-2f9309809fa0ed116cc955a91dc2c3b0R374.

Did you have something else in mind?